docroot/themes/custom/govcms_uikit_starterkit/sass/_config.scss

```diff
@@ -47,6 +47,10 @@ $large: $screen $medium-max + 1;
 $uikit-media-sm: $small-max;
 $uikit-media-md: $medium-max;
 
+$uikit-colour-Hero: #2d5873;
```

Comment:
Intentional the UI kit uses part camel case for their variables and these are to override them.